contracts/modules/staking/StakingPool.sol

```diff
@@ -135,6 +135,8 @@ contract StakingPool is IStakingPool, SolmateERC721 {
   // 1e18
   uint public constant TOKEN_PRECISION = 1 ether;
 
+  uint public constant ALLOCATIONS_DENOMINATOR = 1e16;
```

Comment:
Not pleased with what i did here in terms of naming, also on the read side would need to apply the reverse transform to the number.

@shark0der i know you suggested we apply this 2 decimal storage to all numbers but i didn't get into that because of the scope of this task.